config/projects-prod/iatlas-project.yaml

```diff
@@ -14,6 +14,12 @@ parameters:
     - "{{stack_group_config.tower_viewer_arn_prefix}}/bruno.grande@sagebase.org"
     - "{{stack_group_config.tower_viewer_arn_prefix}}/adam.taylor@sagebase.org"
     - "{{stack_group_config.tower_viewer_arn_prefix}}/james.eddy@sagebase.org"
+    # The following roles don't exist since the users are not Sage employees.
+    # However, this will ensure that they are added to the Tower workspace
+    # until we find a permanent solution for listing external users.
+    - "{{stack_group_config.tower_viewer_arn_prefix}}/benjamin_vincent@med.unc.edu"
+    - "{{stack_group_config.tower_viewer_arn_prefix}}/dbortone@email.unc.edu"
+    - "{{stack_group_config.tower_viewer_arn_prefix}}/steven_vensko@med.unc.edu"
```

Comment:
If these users also need access to the S3 buckets, they will also need to have their actual IAM users/roles added here as well. Also, their Google accounts will need to be added [here](https://github.com/Sage-Bionetworks-Workflows/nextflow-infra/blob/f749c846fb5d59e276a76422ba6cb6c5e3d15932/templates/nextflow-tower-config-bucket.yaml#L72-L74).

Reply:
Sorry, I missed that. If these emails are tied to Google accounts, they'll need to be added to the file that I linked.